src/test/java/core/basesyntax/Car.java

```diff
@@ -45,3 +45,4 @@ public int hashCode() {
         return Objects.hash(model, color);
     }
 }
+
```

Comment:
make sure you don't make unintended changes

Reply:
not fixed